go-controller/pkg/ovn/gateway_init.go

```diff
@@ -159,7 +159,8 @@ func gatewayInit(nodeName string, clusterIPSubnet []*net.IPNet, hostSubnets []*n
 				gatewayProtoLBMap[proto], stderr, err = util.RunOVNNbctl("--", "create",
 					"load_balancer",
 					fmt.Sprintf("external_ids:%s_lb_gateway_router=%s", proto, gatewayRouter),
-					fmt.Sprintf("protocol=%s", strings.ToLower(string(proto))))
+					fmt.Sprintf("protocol=%s", strings.ToLower(string(proto))),
+					"options:hairpin_snat_ip=169.254.169.3 fd69::3")
```

Comment:
Do we need to detect this feature?

Reply:
I dont think we need to detect it (we always assume ovnk is on latest OVN), but we need to consider upgrade here and adding the option if it is missing on these LBs

Reply:
@trozet So what's the best mechanism to do that? On ovnkube-master start should I look through all lbs?

Reply:
I make sure now that before creating any LBs we check if they already exist, then we update the options if needed